libraries/AP_Airspeed/AP_Airspeed.h

```diff
@@ -238,13 +220,38 @@ class AP_Airspeed
     // return the differential pressure in Pascal for the last airspeed reading for the requested instance
     // returns 0 if the sensor is not enabled
     float get_pressure(uint8_t i);
+    // return the current corrected pressure
+    float get_corrected_pressure(uint8_t i) const {
```

Comment:
It's entirely internal usage now, I'd actually nominate that we remove the wrapper, since it's obfuscating what we are actually doing, (the method may be doing lots of work rather then just dereferencing the array). This is an incremental improvement though, so doesn't necessarily need to come in now.